.github/workflows/lint.yml

```diff
@@ -0,0 +1,23 @@
+name: Eslint
+on:
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+    strategy:
```

Comment:
Would be easier to just specify the node version directly in the setup-node action and remove the matrix strategy. We don't need to run this on different node versions.

Reply:
I used the recommended code from the pnpm documentation for this setup. I think I would keep it like that for now and update it if we notice any problems if that's OK

Reply:
It should no cause any problems, it's just an unnecessary complication. But I'm also fine with keeping it like that.